api/apiFunctions.ts

```diff
@@ -87,17 +119,23 @@ export const getNFLTeams = async (): Promise<Models.Document[]> => {
  * @param userId - The user ID
  * @returns {Models.Document | Error} - The game group or an error
  */
-export const getCurrentGame = async (
-  userId: IUser['id'],
-): Promise<Models.Document> => {
+export const getCurrentLeague = async (
+  leagueId: ILeague['leagueId'],
+): Promise<ILeague> => {
   try {
     const response = await databases.listDocuments(
       appwriteConfig.databaseId,
-      '6626a937b6302f6a4d28',
-      [Query.contains('participants', userId)],
+      Collection.LEAGUE,
+      [Query.contains('$id', leagueId)],
     );
 
-    return response.documents[0];
+    return {
+      leagueId: response.documents[0].$id,
```

Comment:
Just like this ID. Save teamId as a reference name.

Reply:
✅ fixed.